deployments/get/defaults.go

```diff
@@ -150,6 +150,48 @@ var defaultConfigs = map[string]DaemonSetConfig{
 			},
 		},
 	},
+	"cri-o": {
```

Comment:
@Ankurk99 @nyrahul we should target `oke` here, right? So that deployment manifests for OpenShift Kubernetes can be generated? :thinking:

Reply:
>we should target oke here

@vikasvr I think it would make more sense. Please also update the main.go environments here: https://github.com/kubearmor/KubeArmor/blob/37002bf115c2a614fe8b643c8eb249ac9a98faf2/deployments/main.go#L23

Reply:
@DelusionalOptimist  Since It's a map of container runtime and associated configuration, I added `cri-o`, is it ok to have provider specific info `oke`.